boards/FSM/Src/Io/Io_Coolant.c

```diff
@@ -25,8 +23,27 @@ void Io_FlowMeters_InputCaptureCallback(TIM_HandleTypeDef *htim)
         Io_SharedFreqOnlyPwmInput_Tick(flow_meter);
     }
 }
-
 void Io_FlowMeters_CheckIfFlowMeterIsActive(void)
 {
     Io_SharedFreqOnlyPwmInput_CheckIfPwmIsActive(flow_meter);
 }
+
+//==========TEMPERATURE SENSORS==========
+float Io_GetTemperatureA(void)
```

Comment:
These are thermistor sensors which mech can move around the coolant loop to collect data. Because we have not received details from mech, this is about all he can do for now, but they will be analog sensors